meeting-notes/2024-01-10.md

```diff
@@ -0,0 +1,31 @@
+# SRT meeting 2024-01-11
```

Comment:
```suggestion
# SRT meeting 2024-01-10
```

Reply:
Thanks; updated.  For me it was the 11th :sweat_smile: